### DISCUSSION TITLE
Self-hosted instance on coolify facing routing issues

### DISCUSSION THREAD

**Discussion by Abz2604** [OpnForm/OpnForm#401] [Q&A]
The logs constantly show the following:

[Vue Router warn]: No match found for location with path "/api/register"
[Vue Router warn]: No match found for location with path "/api/register"
[Vue Router warn]: No match found for location with path "/api/register"
[Vue Router warn]: No match found for location with path "/api/register"

I used the docker image available directly from the docker registry (jhumanj//opnform)
The only change I made was the port exposed.

The homepage opens up as expected
But when we go ahead and try to register we face this issue. The create account button basically doesnt respond and we get the Vue Routing error in the logs

**Comment by JhumanJ** [comment]
Coolify's template available here https://github.com/coollabsio/coolify/pull/5875
